gramex/ml_api.py

```diff
@@ -46,6 +46,10 @@
         "statsmodels.tsa.statespace.sarimax",
     ],
     "gramex.ml_api.HFTransformer": ["gramex.transformers"],
+    "gramex.ml_api.KerasApplications": [
+        "tensorflow.keras.applications.vgg16",
+        "tensorflow.keras.applications.resnet50"
```

Comment:
You can just leave this at `tensorflow.keras.applications`, because
```python
from tensorflow.keras.applications import *
```
covers everything